.circleci/config.yml

```diff
@@ -331,6 +331,11 @@ jobs:
             elif [ "$CIRCLE_BRANCH" == "raft-tdp-main" ] ; then
               cd tdrs-frontend; bash <(curl -s https://codecov.io/bash)  -f ./coverage/lcov.info -F dev-frontend
             fi
+      - run:
+          name: "Install lib bindings for Cypress"
```

Comment:
This additional package was needed to get Cypress to work on version 7.7: [ref](https://docs.cypress.io/guides/continuous-integration/introduction#Dependencies)

Reply:
Note: We are on v8 now- there was a new version of Cypress available.